src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (a, b) => {
+      return a + b;
+    },
+
+    subtract: (a, b) => a - b,
+
+    multiply: (a, b) => a * b,
+
+    divide: (a, b) => {
+      if (b === 0) {
+        return a;
```

Comment:
The current implementation of the `divide` function returns `a` when `b` is zero. This might not be the desired behavior for division by zero. Consider throwing an error or returning a specific value to indicate that the operation is invalid.